config/locales/cy.yml

```diff
@@ -69,6 +69,10 @@ cy:
       employer_validation:
         blank: Rhowch gyfraniad eich cyflogwr
         out_of_range: Rhaid i gyfraniad eich cyflogwr fod yn yr ystod 0 - 100%
+      manually_optIn: |
```

Comment:
I think the best name for this should be `manually_opt_in`. The first impression of `manually_optIn` is that it's not terribly clear and it looks like `optIn` is a misspelling of _option_.